code/frameworks/nextjs/package.json

```diff
@@ -47,6 +47,11 @@
       "require": "./dist/next-image-loader-stub.js",
       "import": "./dist/next-image-loader-stub.mjs"
     },
+    "./headers": {
```

Comment:
Crazy idea, what if the entrypoitn was `@storybook/nextjs/headers.mock` instead? To more clearly communicate what it does and to follow the usual pattern of `import X from './foo.mock`.

Just an idea, no strong feeling.

Reply:
I think I like it. 🤔